docker-compose.yml

```diff
@@ -0,0 +1,9 @@
+version: "3"
+services:
+  site:
+    command: jekyll serve --watch
+    image: jekyll/jekyll:latest
```

Comment:
Good idea! Do we want to lock to a particular version of the jekyll container?

Reply:
We could.  I'm not sure what version to use.  I'm not a regular jekyll user.